src/openMVG/multiview/rotation_averaging_l1.cpp

```diff
@@ -329,7 +329,7 @@ struct Link {
 };
 typedef std::queue<Link> LinkQue;
 
-#ifdef HAVE_BOOST
+#ifdef OPENMVG_ROTATION_AVERAGING_WITH_BOOST
```

Comment:
Not sure if we want to make that optional.
It's only there to keep the code AFAIK.
I just changed it to a dedicated macro, because there are other reasons to build with boost without using this rotation averaging.